spec/requests/bridges_request_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require './spec/support/main_helper'
+
+RSpec.configure do |c|
+  c.include MainHelper
+end
+
+RSpec.describe 'Bridges', type: :request do
+  before do
+    create_user
+  end
+
+  after do
+    @current_user.destroy
+  end
+
+  subject do
+    create_bridge
+  end
+
+  describe 'handles all requests properly:' do
+    it 'index method' do
+      subject.name = 'index method bridge'
+      subject.save!
+      get bridges_path, headers: authenticated_token
+      expect(response).to be_successful
```

Comment:
Again testing only perfect world. Where are the tests for testing bad data? Do you handle errors properly?